history/characters/310000_patron_characters.txt

```diff
@@ -835,4 +835,104 @@
 		religion=scarlet_light
 	}
 	648.1.5={ death=yes }
+}
+
+#dynasty=125040
+310400={ #Count #Spartan
+	name=Edward
+	dynasty=125040
+	dna="0djfj0f0ab0"
+	properties="0l00fk0000000000000000"
+	culture=gilnean religion=holy_light
+	martial=5 diplomacy=5 stewardship=5 intrigue=5 learning=5
+	effect={add_patron_trait_character_effect=yes}
+	disallow_random_traits = yes
+	559.12.15={ birth=yes add_trait=creature_human }
+	580.1.1={
+		add_trait=intricate_webweaver
+		add_trait=class_rogue_6
+		add_trait=shrewd
+		add_trait=strategist
+		add_trait=falconer add_trait=ambitious add_trait=proud add_trait=stubborn add_trait=greedy add_trait=envious add_trait=cynical add_trait=patient add_trait=ravager
+	}
+	612.10.28={ death=yes }
+}
+
+#dynasty=125041
+310410={ #Count #Daruthyr
+	name=Gyliad
+	dynasty=125041
+	dna="00ldb0b0eb0"
+	properties="0d00df0000000000000000"
+	culture=arathorian religion=holy_light secret_religion=old_ways
+	martial=0 diplomacy=5 stewardship=5 intrigue=5 learning=5
+	effect={add_patron_trait_character_effect=yes}
+	add_trait=legit_bastard add_trait=twin add_trait=lefthanded
+	add_trait=fair
+	disallow_random_traits = yes
+	560.1.27={ birth=yes add_trait=creature_human }
+	580.1.1={
+		add_trait=brilliant_strategist
+		add_trait=class_warrior_6
+		add_trait=mystic
+		add_trait=scarred_mid add_trait=one_eye add_trait=lustful add_trait=wroth add_trait=brave add_trait=paranoid add_trait=stubborn add_trait=zealous add_trait=unyielding_leader add_trait=rough_terrain_leader
```

Comment:
one_eyed, not one_eye :)

Reply:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/f37100e301f5cb38789cb7f76506f9d0e4a4e718.